quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/QuarkusPlatform.java

```diff
@@ -159,4 +166,48 @@ public File getTmpDirectory() {
     private void reset() {
         deferredExceptions.clear();
     }
+
+    @Override
+    public ClassLoader getScriptEngineClassLoader(Config.Scope scriptProviderConfig) {
```

Comment:
In Quarkus we have a flat classpath so I'm unsure if we need to create a classloader?

Reply:
@pedroigor You're right, Thanks for this point! After some digging, I figured that it is completely sufficient to just include the nashorn jar and it's dependencies directly inside "providers" directory. They are simply picked from there by the default flat classloader. So I changed the Quarkus part to just return null, which is simply using the same default (flat) classloader.